drivers/display/display_otm8009a.c

```diff
@@ -0,0 +1,872 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
```

Comment:
-1 on such poor quality driver. I actually had an incomplete PoC for such display, see https://github.com/teslabs/zephyr/commits/feature/stm32-dsi. You are welcome to re-use it if it works for you.

Reply:
If I get it right, you suggest to rewrite the OTM8009A driver from your source?

Reply:
No need to take that source, just provide a native driver instead of pulling poor quality vendor code.

Reply:
Is it still poor qualitiy if this commit https://github.com/zephyrproject-rtos/zephyr/pull/53044/commits/3d9d4b351b64b7a73828b8a2331a1349efd3c4b2 would be dropped (squashed)?

Reply:
While it looks better, there's still some "vendor magic" that should be improved. Also, we don't need to keep vendor copyrights when doing a native driver. Your latest version is closer to native, so I think it's worth the effort to polish it. I guess you can merge some code from  https://github.com/teslabs/zephyr/commit/ecaa38c089829ed2883e33955bababc28da9c994 (e.g. rotation support, register definitions, etc.)